src/main/java/teammates/ui/controller/InstructorHomePageAction.java

```diff
@@ -36,10 +33,11 @@ public ActionResult execute() throws EntityDoesNotExistException {
         }
         
         boolean omitArchived = true;
-        HashMap<String, CourseSummaryBundle> courses = logic.getCourseSummariesWithoutStatsForInstructor(account.googleId, omitArchived);
-        
+        HashMap<String, CourseSummaryBundle> courses = logic.getCourseSummariesWithoutStatsForInstructor(
+                                        account.googleId, omitArchived);
```

Comment:
Indent further so that the 2nd line appears on the right of the `logic.`. When possible, try to have the child element on the right of the parent element.